src/domains/transaction/components/TransactionTable/PendingTransactionsTable/PendingTransactionsTable.tsx

```diff
@@ -24,14 +25,15 @@ export const PendingTransactions = ({
 	const { t } = useTranslation();
 	const [pendingRemovalTransaction, setPendingRemovalTransaction] = useState<DTO.ExtendedSignedTransactionData>();
 
-	const columns = usePendingTransactionTableColumns();
+	const columns = usePendingTransactionTableColumns({ coin: wallet.network().coinName() });
 
 	const { isXs, isSm } = useBreakpoint();
 
 	const useResponsive = useMemo(() => isXs || isSm, [isXs, isSm]);
 
 	const renderTableRow = useCallback(
 		(transaction: PendingTransaction) => {
+			/* NOTE: Pending transfer refers to the status of the transaction pending validation in the chain and not related to musig wallet */
```

Comment:
Added this comment as a hint for devs, since it was a bit confusing to treat the different "Pending" transactions.